src/main/webapp/app/admin/dashboard/admin-dashboard.component.ts

```diff
@@ -0,0 +1,86 @@
+import { defineComponent, inject, ref, type Ref } from 'vue';
+
+import type { ILocation } from '@/shared/model/location.model';
+import LocationService from '@/entities/location/location.service';
+import type { IAdminChecklist } from '@/shared/model/admin-checklist.model';
+import AdminDashboardService from '@/admin/dashboard/admin-dashboard.service';
+import axios from 'axios';
+
+export default defineComponent({
+  compatConfig: { MODE: 3 },
+  name: 'AdminDashboard',
+  setup() {
+    const locationService = inject('locationService', () => new LocationService());
+    const adminDashboardService = inject('adminDashboardService', () => new AdminDashboardService());
+
+    const locations: Ref<ILocation[]> = ref([]);
+    const checklists: Ref<IAdminChecklist[]> = ref([]);
+    const bccForAllUsers: Ref<string> = ref('');
+    const countAllProfiles: Ref<number> = ref(0);
+    const bccForAllUsersWithUncompleteProfile: Ref<string> = ref('');
```

Comment:
Please name it `bccForAllUsersWithIncompleteProfile` (Uncomplete -> Incomplete).